engine/server/engine/centralized_logs/client_implementations/persistent_volume/log_remover/log_remover.go

```diff
@@ -0,0 +1,44 @@
+package log_remover
+
+import (
+	"fmt"
+	"github.com/kurtosis-tech/kurtosis/engine/server/engine/centralized_logs/client_implementations/persistent_volume/logs_clock"
+	"github.com/kurtosis-tech/kurtosis/engine/server/engine/centralized_logs/client_implementations/persistent_volume/volume_consts"
+	"github.com/kurtosis-tech/kurtosis/engine/server/engine/centralized_logs/client_implementations/persistent_volume/volume_filesystem"
+	"github.com/sirupsen/logrus"
+	"strconv"
+	"time"
+)
+
+const (
+	oneWeekDuration = 7 * 24 * time.Hour
```

Comment:
nit: You variable is already of type "duration" here, no need to add it to its name